vivisect/impapi/windows/arm.py

```diff
@@ -0,0 +1,22 @@
+import vivisect.impapi.windows.i386 as v_w_i386
+apitypes = dict(v_w_i386.apitypes)
```

Comment:
kinda odd that we have to have this interdependency, but it's not harmful I think

Reply:
the api's don't change (much?) between architectures.  just the calling conventions.  this allows us to only change any specific function api that actually changes.